api/management/commands/generate_fake_actions.py

```diff
@@ -24,8 +24,8 @@ def handle(self, *args, **options):
             coop = Cooperative.objects.get(pk=options['coop_id'])
 
             for n in range(options['quantity']):
-                action = Action(date=datetime.date.today(), name="Action {}".format(n),
-                                description="Action {} description".format(n), invested_money=random.choice(investments),
+                action = Action(date=datetime.date.today(), name=f"Action {n}",
```

Comment:
This is not `py2-like` string format? Why it needed to change from `.format(..)`?

Reply:
This is Python 3's f-Strings, IMO it's more readable and less prone to error than other ways of formatting (because of the distance between information a place where should be).